src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -54,7 +97,16 @@ Anomaly detection uses the following methods to detect anomalies in your app dat
 
 Applications will not always generate anomalies, so it can be normal to not receive any detections.
 
-## Set up notifications for anomaly detection [#set-up]
+### Requirements [#requirements]
+
+To use New Relic's anomaly detection, ensure you have:
+
+* An APM agent installed for at least one application.
+* To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).
+
+For more on data limits, see [Data limits](#data-limits).
+
+### Set up notifications for anomaly detection [#set-up]
 
 Proactive detection is enabled automatically, at no additional cost. To receive notifications or to have a configuration (group of apps) that you can add as a source for incident intelligence, you will need to create a anomaly detection configuration. You can create a configuration in the anomaly detection UI:
```

Comment:
replaces proactive detection with "automatic anomaly detection".  May want to do a quick search for proactive detection across all our docs and replace with "automatic anomaly detection".

Reply:
done!